plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlConnectorTest.java

```diff
@@ -298,6 +299,14 @@ public void testPredicatePushdown()
                 .isNotFullyPushedDown(AggregationNode.class);
     }
 
+    @Test
+    @Override
+    public void testInsertNegativeDate()
+    {
+        // TODO (https://github.com/trinodb/trino/issues/10208) MemSQL stores '0000-00-00' when inserted negative dates and it throws an exception during reading the row
+        assertThatThrownBy(super::testInsertNegativeDate);
```

Comment:
as above, assertThatThrownBy should be followed by an assertion over the exception.